test/tangent_types/tangent.jl

```diff
@@ -148,12 +148,16 @@ end
         cr = Tangent{Tuple{String,Int,Int}}("something", 2, 1)
         @test reverse(c) === cr
 
-        # can't reverse a named tuple or a dict
-        @test_throws MethodError reverse(Tangent{Foo}(; x=1.0, y=2.0))
+        if VERSION < v"1.9-"
+            # can't reverse a named tuple or a dict
+            @test_throws MethodError reverse(Tangent{Foo}(; x=1.0, y=2.0))
```

Comment:
Likewise needed for 1.9

This tests that you can't reverse a Tangent, but do we care that you *can't*? We know that it's meaningless, like the order of fields in a struct. But if it used to be an error and now works, why is that a failure?